tests/dragonfly/seeder/script-genlib.lua

```diff
@@ -1,9 +1,43 @@
 local LG_funcs = {}
 
-function LG_funcs.init(dsize, csize)
+function LG_funcs.init(dsize, csize, large_val_perc, large_val_sz)
     LG_funcs.dsize = dsize
     LG_funcs.csize = csize
     LG_funcs.esize = math.ceil(dsize / csize)
+    LG_funcs.huge_value_percentage = large_val_perc
+    LG_funcs.huge_value_size = large_val_sz
+end
+
+local huge_entries = 0
+
+local function huge_entry()
```

Comment:
Don't you want `huge_entry()` to depend on the key? Such that some keys are huge, while others aren't, based on (say) their hash?
The reason I say this is because the seeder uses many operations to generate the values, like many `lpush`, `hset`, etc. If we do 100 operations per key (just throwing numbers here), doing 5% huge will make them all roughly of the same size...